mkdocs.yml

```diff
@@ -59,13 +58,20 @@ nav:
       - Connect to Wi-Fi: setup/provision.md
       - Multi-Robot Setup: setup/multi-robot.md
       - ROS 2 Network Config: setup/xml-config.md
-      - Setup Compute Boards:
+      - Connected Computers:
         - ROS 2 on Ubuntu 20.04: setup/ubuntu2004.md
-        - ROS 2 on a NavQ+: setup/navqplus.md
-        - ROS 2 on a Pi 4: setup/pi4.md
-        - ROS 2 on an NVIDIA Jetson: setup/jetson.md
-        - NTP on Compute Board: setup/compute-ntp.md
         - Desktop Docker: setup/docker.md
+      - Compute Boards: 
```

Comment:
maybe it would be worth saying "Attached Compute Boards" just to be more clear? Interested in others' feedback.

Reply:
Hmmm, I would lean more towards "Attached Compute" maybe even dropping the boards from the name.

In general my preference is more succinct navigation items, I feel like "third-party" might be an overly verbose designation to something that the end user already realizes is third-party. But I think any addition of those descriptors is fine.